Makefile

```
@@ -14,7 +14,7 @@ all: test build
 
 build: builddir clean npm frontend sippy sippy-daemon
 
-verify: lint
+verify: lint verify-apm
```

checkmake (0.3.2)

[warning] Target "verify" should be declared PHONY.

(phonydeclared)